src/topology/algebra/infinite_sum.lean

```diff
@@ -35,9 +35,10 @@ section has_sum
 variables [add_comm_monoid α] [topological_space α]
 
 /-- Infinite sum on a topological monoid
-The `at_top` filter on `finset α` is the limit of all finite sets towards the entire type. So we sum
-up bigger and bigger sets. This sum operation is still invariant under reordering, and a absolute
-sum operator.
+The `at_top` filter on `finset β` is the limit of all finite sets towards the entire type. So we sum
+up bigger and bigger sets. This sum operation is invariant under reordering. In particular,
+the function `ℕ → ℝ` sending n to (-1)^n/(n+1) does not have a
+sum for this definition, but a series which is absolutely convergent will have the correct sum.
 
 This is based on Mario Carneiro's infinite sum in Metamath.
```

Comment:
```suggestion
This is based on Mario Carneiro's 
[infinite sum `df-sum` in Metamath](http://us.metamath.org/mpeuni/df-sum.html).
```
@digama0 is this the right definition / link?

Reply:
I committed the suggestion and it marked the comment as resolved; I marked it as unresolved and now Bryan's suggestion is still there -- but just to be clear, Bryan's suggestion has been committed.

Reply:
No, it is http://us.metamath.org/mpeuni/df-tsms.html